src/spikeinterface/curation/curationsorting.py

```diff
@@ -59,9 +61,29 @@ def _get_unused_id(self, n=1):
             ids = [str(i) for i in ids]
         return ids
 
-    def split(self, split_unit_id, indices_list):
+    def split(self, split_unit_id, indices_list, new_unit_ids=None):
+        """
+        Split a unit into multiple units.
+
+        Parameters
+        ----------
+        split_unit_id: int
```

Comment:
can this also be string? since the ids can be strings or ints?

Reply:
done, but this is not a list so Iused `int or str`

Reply:
done